File: Golang/657-robot-return-to-origin/robot-return-to-origin.go
func judgeCircle(moves string) bool {
    X, Y := 0, 0
    for _, char := range moves {
        if char == 'U' {
            Y++
        } else if char == 'D' {
            Y--
        } else if char == 'R' {
            X++
        } else {
            X--
        }
    }
    // fmt.Println("x = ",X,"\ny = ",Y)
    if X == 0 && Y == 0 {
        return true
    }
    return false
}

File: Python/657-robot-return-to-origin/robot-return-to-origin.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        X = Y = 0
        for char in moves:
            if char == 'U':
                Y += 1
            elif char == 'D':
                Y -= 1
            elif char == 'R':
                X += 1
            elif char == 'L':
                X -= 1
        if X == 0 and Y == 0:
            return True
        else:
            return False